back/engines/commercial/multi_tenancy/lib/tasks/core/migrate_votes_to_reactions.rake

```diff
@@ -0,0 +1,138 @@
+# frozen_string_literal: true
+
+# NOTE: Only to be used for release that migrates Votes -> Reactions
+namespace :fix_existing_tenants do
+  desc 'Transform all core data relating to votes into reactions'
+  task migrate_votes_core: [:environment] do |_t, _args|
+    # Order by active first - to reduce downtime
+    tenants = Tenant.creation_finalized.with_lifecycle('active') + Tenant.creation_finalized.with_lifecycle('demo')
```

Comment:
This will skip other lifecycle stages than active and demo? Like churned, trial, expired_trial and not_applicable.